widgetastic_manageiq.py

```diff
@@ -1108,3 +1109,27 @@ def select(self, title):
     @property
     def selected(self):
         return next(btn.title for btn in self._view_buttons if btn.active)
+
+
+class Search(View):
+    """ Represents search_text control
+    # TODO Add advanced search
+    """
+    search_text = Input(name="search_text")
+    search_btn = Text("//div[@id='searchbox']//div[contains(@class, 'form-group')]"
+                      "/*[self::a or (self::button and @type='submit')]")
+    clear_btn = Text(".//*[@id='searchbox']//div[contains(@class, 'clear')"
+                     "and not(contains(@style, 'display: none'))]/div/button")
+
+    def clear_search(self):
+        if not self.is_empty:
+            self.clear_btn.click()
+            self.search_btn.click()
+
+    def search(self, text):
+        self.search_text.fill(text)
+        self.search_btn.click()
+
+    @property
+    def is_empty(self):
+        return self.search_text.read() == ''
```

Comment:
**OPTIONAL**: `return bool(self.search_text.text)`

Empty string resolves to a bool and using .text directly avoids the read logging. You might like to add a logging call in this property then:
```python
@property
def is_empty(self):
    empty = bool(self.search_text.text)
    self.logger.info('checked if empty: %r', empty)
    return empty
```

Or also a possibility:
```python
from widgetastic.log import logged

@property
@logged(log_result=True)
def is_empty(self):
    return bool(self.search_text.text)
```